test/integration/services/lnd/health.spec.ts

```diff
@@ -0,0 +1,26 @@
+import moment from "moment"
+import { isUp, lndStatusEvent } from "@services/lnd/health"
+import { params } from "@services/lnd/unauth"
+import { baseLogger } from "@services/logger"
+
+beforeAll(done => {
+  done()
+})
+
+afterAll(done => {
+  done()
+})
+
+describe("lndHealth", () => {
+  // this is a test health checks on lnd
+  it('should emit on started', function(){
+    let eventFired = true
```

Comment:
Looks like this variable is initialised to `true` so the test doesn't actually test anything.